pom.xml

```diff
@@ -660,73 +660,45 @@
       <id>run-code-coverage</id>
       <properties>
         <jacoco.excludes>*Lexer</jacoco.excludes>
-        <jacoco.exec.file>${project.build.directory}/jacoco.exec</jacoco.exec.file>
-      </properties>
 
-      <dependencyManagement>
-        <dependencies>
-          <dependency>
-            <groupId>org.jacoco</groupId>
-            <artifactId>org.jacoco.agent</artifactId>
-            <classifier>runtime</classifier>
-            <version>${version.jacoco.plugin}</version>
-            <scope>test</scope>
-          </dependency>
-        </dependencies>
-      </dependencyManagement>
-      <dependencies>
-        <dependency>
-          <groupId>org.jacoco</groupId>
-          <artifactId>org.jacoco.agent</artifactId>
-          <classifier>runtime</classifier>
-          <scope>test</scope>
-        </dependency>
-      </dependencies>
+        <!-- jacoco.exec.file is defined in kie-parent -->
+        <!--suppress UnresolvedMavenProperty -->
```

Comment:
There should be a space between "<!--" and "suppress". Yes, I committed the same crime in some other repositories.